main.ts

```diff
@@ -3,9 +3,9 @@
 
 require("dotenv").config();
```

Comment:
@FraWolf 
In genere non son un grande fan di `dotenv` usato così, soprattutto se l'intenzione è quella di usare docker.
Per ora lo lascio ma volendo si può usare solo per sviluppare (basta configurare nodemon) e usare le variabili d'ambiente di docker in produzione

Reply:
Tu cosa consigli di fare? Il .env io lo facevo per praticità per la persona

Reply:
Si può usare questo script per lanciare l'app mentre si sviluppa, in questo modo viene caricato `dotenv` al di fuori dell'app, diciamo.
```
node -r dotenv/config index.js
```
Poi in prod si builda l'immagine docker e si lancia l'app normalmente (con `node index.js`) usando le variabili d'ambiente di docker.

Reply:
Sisi, ma io dicevo per far configurare il bot, sia per il dev che per il prod

Reply:
Il file `.env` resta per il dev ma su prod si usa questo https://docs.docker.com/engine/reference/run/#env-environment-variables
È meglio tenerle proprio come due cose separate secondo me.

Reply:
Comunque è slegato da questa PR, poi ne parliamo meglio

Reply:
Yup